### ISSUE TITLE
OBJ to CGFX odd glich

### ISSUE THREAD

**Issue by quizque** [Gericom/EveryFileExplorer#26]
I'm trying to make a tutorial on how to make 3D banners for CIA homebrews but when I convert the obj to CGFX this happens 
![capture](https://cloud.githubusercontent.com/assets/13785390/15991447/12e28438-3081-11e6-84c4-55c4719b3ffe.PNG)
I don't know how to fix it and no one else who I have talked to have had this problem, I have opened it in multiple programs, I also use Wings3D to model and export as obj.

**Comment by Gericom** [comment]
Are you sure your model is fully triangulated?

**Comment by quizque** [comment]
Yes they are, I tried it with just a cube and it didn't work but when I open it in another modelling program like blender it's fine it's only EFE that chops up the model :/

**Comment by dixy52-beep** [comment]
you have to check the "triangulate" checkbox in export settings, not the modifier
